ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_QMLSamples/android/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <manifest package="com.esri.arcgisruntime.ArcGISRuntimeSDKQt_QMLSamples" xmlns:android="http://schemas.android.com/apk/res/android" android:versionName="-- %%INSERT_VERSION_NAME%% --" android:versionCode="-- %%INSERT_VERSION_CODE%% --" android:installLocation="auto">
-    <uses-sdk android:minSdkVersion="23" android:targetSdkVersion="23"/>
+    <uses-sdk android:minSdkVersion="26" android:targetSdkVersion="26"/>
```

Comment:
FYI this may have permissions fallout, so keep this in mind if anything strange starts happening in the sample viewers on Android.

Reply:
cc @tanneryould - I know your current work touches the manifest, so please incorporate these changes